backend/controllers/caseNote.js

```diff
@@ -137,6 +137,24 @@ const caseNoteFactory = (elite2Api, caseNotesApi) => {
       })
     }
 
+    try {
+      const dateTime = moment(date, 'DD/MM/YYYY')
+        .hours(hours)
+        .minutes(minutes)
+        .seconds(0)
+        .format('YYYY-MM-DDTHH:mm:ss')
+
+      if (dateTime > moment().format('YYYY-MM-DDTHH:mm:ss')) {
+        errors.push({
+          text: 'Enter a time which is not in the future',
+          href: '#hours',
+        })
+      }
+    } catch {
+      // Do nothing, will only fail if the date inputs are invalid, which
+      // has been handled by the above validations
+    }
```

Comment:
There's no need to do anything in the catch, if the date creation failed it would be because one of the validation errors covered above.